Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+        private const string TabContentClassName = "TabContent";
+
+        private static GameLiftPlugin GetWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
+            var window = GetWindow<GameLiftPlugin>(inspectorType);
+            window.titleContent = new GUIContent("Amazon GameLift", window.icon);
+            return window;
+        }
+
+        [MenuItem("Amazon GameLift/Show Amazon GameLift Window", priority = 0)]
+        public static void ShowWindow()
+        {
+            GetWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetWindow().OpenTab("Credentials");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Anywhere", priority = 101)]
+        public static void OpenAnywhereTab()
+        {
+            GetWindow().OpenTab("Anywhere");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Managed EC2", priority = 102)]
+        public static void OpenEC2Tab()
+        {
+            GetWindow().OpenTab("EC2");
+        }
+
+        [MenuItem("Amazon GameLift/Import Sample Game", priority = 103)]
+        public static void ImportSampleGame()
+        {
+            string filePackagePath = $"Packages/{Paths.PackageName}/{Paths.SampleGameInPackage}";
+            AssetDatabase.ImportPackage(filePackagePath, interactive: true);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Documentation", priority = 200)]
+        public static void OpenDocumentation()
+        {
+            Application.OpenURL(Urls.AwsHelpGameLiftUnity);
+        }
+
+        [MenuItem("Amazon GameLift/Help/AWS GameTech Forum", priority = 201)]
+        public static void OpenGameTechForums()
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            
+            ApplyText();
+
+            var contentContainer = _root.Q(className: "main__content");
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
```

Comment:
We should write reusable utilities to perform queries like these, that can be reused across different visual components.

Reply:
I'm not sure what you are requesting with this specific code highlight.

Reply:
My bad, I assumed _root.Query was not the conventional way of accessing elements, but looks like that's the way to go about referencing UX elements while using UXML. Is that understanding right? 

I see usages of `_root.Q` and `_root.Query`. Are these functionally the same? If so, can we be consistent and stick to `Query`

Reply:
These are not functionally the same. `.Q` is equivalent to `.Query().First()`